### ISSUE TITLE
How to use tensorflow?

### ISSUE THREAD

**Issue by panhu** [echocatzh/conv-stft#1]
Hi,if i want to use tensorflow,how to change?

def init_kernels(win_len, win_inc, fft_len, win_type=None, invers=False):
    if win_type == 'None' or win_type is None:
        window = np.ones(win_len)
    else:
        window = get_window(win_type, win_len, fftbins=True)**0.5
    
    N = fft_len
    fourier_basis = np.fft.rfft(np.eye(N))[:win_len]
    real_kernel = np.real(fourier_basis)
    imag_kernel = np.imag(fourier_basis)
    kernel = np.concatenate([real_kernel, imag_kernel], 1).T
    
    if invers :
        kernel = np.linalg.pinv(kernel).T 

    kernel = kernel*window
    kernel = kernel[:, None, :]
    return tf.convert_to_tensor(kernel,tf.float32)

class ConvSTFT(tf.keras.layers.Layer):

    def __init__(self, win_len, win_inc, fft_len, win_type='hamming', feature_type='real', fix=True):
        super(ConvSTFT, self).__init__() 
        
        self.fft_len = fft_len
        
        kernel= init_kernels(win_len, win_inc, self.fft_len, win_type)
        self.weight = tf.Variable(kernel)
        self.feature_type = feature_type
        self.stride = win_inc
        self.win_len = win_len
        self.dim = self.fft_len

    def call(self, inputs):

        outputs = tf.nn.conv1d(inputs, self.weight, stride=self.stride,padding='VALID')
         
        output_list = []
        
        dim = self.dim//2+1
        real = outputs[:, :dim, :]
        imag = outputs[:, dim:, :]
        output_list = [real,imag]
        return output_list

It is right?

**Comment by echocatzh** [comment]
Sorry, I'm not sure if it works with `tensorflow`, you can compare the outputs with `librosa.stft` if `pytorch` is not available.
